src/teehr/examples/07-2_site_plotting.ipynb

```diff
@@ -31,6 +31,77 @@
     "ev.enable_logging()"
    ]
   },
+  {
```

Comment:
I got errors when running this notebook due to missing tables, I guess we need to run `setup_evaluation.py` first?

Reply:
Yeah -- I had sort of just been using this notebook for dynamic testing/debugging prior to updating the official test case. I imagine it will sequentially follow the existing notebook examples and the content in this example will evolve as the class shapes up.